w_titanic

```diff
@@ -0,0 +1,12 @@
+library(tidyr)
+library(dplyr)
+
+heart <- read.csv("titanic_original.csv", header = TRUE, stringsAsFactors = FALSE)
+
+heart$embarked[which(is.na(heart$embarked))] <- "S"
```

Comment:
I don't think that the entries you need to fix are na; I think they are blank. So instead of:
    is.na(heart$embarked)

you should use:
    heart$embarked == ''